WeakAurasTemplates/TriggerTemplatesDataMists.lua

```diff
@@ -187,21 +186,107 @@ templates.class.PALADIN = {
+        { spell = 642, type = "ability", buff = true }, -- Divine Shield
+        { spell = 853, type = "ability", debuff = true, requiresTarget = true }, -- Hammer of Justice
+        { spell = 879, type = "ability", overlayGlow = true }, -- Exorcism
+        { spell = 1022, type = "ability", charges = true, buff = true, usable = true }, -- Hand of Protection
+        { spell = 1038, type = "ability", charges = true, buff = true }, -- Hand of Salvation
+        { spell = 1044, type = "ability", charges = true, buff = true }, -- Hand of Freedom
+        { spell = 2812, type = "ability", debuff = true, requiresTarget = true }, -- Denounce
+        { spell = 4987, type = "ability" }, -- Cleanse
+        { spell = 6940, type = "ability", charges = true }, -- Hand of Sacrifice
+        { spell = 10326, type = "ability" }, -- Turn Evil
+        { spell = 20066, type = "ability", talent = 5 }, -- Repentance
+        { spell = 20271, type = "ability", requiresTarget = true, usable = true }, -- Judgment
+        { spell = 20473, type = "ability" }, -- Holy Shock
+        { spell = 20925, type = "ability", buff = true, talent = 9 }, -- Sacred Shield
+        { spell = 24275, type = "ability", overlayGlow = true, requiresTarget = true, usable = true }, -- Hammer of Wrath
+        { spell = 25780, type = "ability", buff = true }, -- Righteous Fury
+        { spell = 26573, type = "ability", totem = true }, -- Consecration
+        { spell = 31821, type = "ability", buff = true }, -- Devotion Aura
+        { spell = 31842, type = "ability", buff = true }, -- Divine Favor
+        { spell = 31850, type = "ability", buff = true }, -- Ardent Defender
+        { spell = 31884, type = "ability", buff = true }, -- Avenging Wrath
+        { spell = 31935, type = "ability", debuff = true, usable = true }, -- Avenger's Shield
+        { spell = 35395, type = "ability", requiresTarget = true, usable = true }, -- Crusader Strike
+        { spell = 53563, type = "ability", buff = true }, -- Beacon of Light
+        { spell = 53595, type = "ability", requiresTarget = true, usable = true }, -- Hammer of the Righteous
+        { spell = 53600, type = "ability", requiresTarget = true, usable = true }, -- Shield of the Righteous
+        { spell = 54428, type = "ability", buff = true }, -- Divine Plea
+        { spell = 62124, type = "ability", debuff = true, requiresTarget = true }, -- Reckoning
+        { spell = 85256, type = "ability", overlayGlow = true, requiresTarget = true }, -- Templar's Verdict
+        { spell = 85499, type = "ability", buff = true, talent = 1 }, -- Speed of Light
+        { spell = 85673, type = "ability", overlayGlow = true }, -- Word of Glory
+        { spell = 86659, type = "ability", buff = true }, -- Guardian of Ancient Kings
+        { spell = 86669, type = "ability", buff = true }, -- Guardian of Ancient Kings
+        { spell = 86698, type = "ability", buff = true }, -- Guardian of Ancient Kings
```

Comment:
can remove 2

Reply:
Will the templates automatically handle the other two specs' GoAKs then?